src/sentry/templatetags/sentry_helpers.py

```diff
@@ -65,10 +65,58 @@ def coerce(value):
     return coerce(x) * coerce(y)
 
 
-@register.simple_tag
-def absolute_uri(path='', *args):
-    from sentry.utils.http import absolute_uri
-    return absolute_uri(path.format(*args))
+class AbsoluteUriNode(template.Node):
+    def __init__(self, args, target_var):
+        self.args = args
+        self.target_var = target_var
+
+    def render(self, context):
+        from sentry.utils.http import absolute_uri
+
+        # Attempt to resolve all arguments into actual variables.
+        # This converts a value such as `"foo"` into the string `foo`
+        # and will look up a value such as `foo` from the context as
+        # the variable `foo`. If the variable does not exist, silently
+        # resolve as an empty string, which matches the behavior
+        # `SimpleTagNode`
+        args = []
+        for arg in self.args:
+            try:
+                arg = template.Variable(arg).resolve(context)
+            except template.VariableDoesNotExist:
+                arg = ''
+            args.append(arg)
+
+        # No args is just fine
+        if not args:
+            rv = ''
+        # If there's only 1 argument, there's nothing to format
+        elif len(args) == 1:
+            rv = args[0]
+        else:
+            rv = args[0].format(*args[1:])
```

Comment:
I'm assuming you checked to ensure that `args[0]` is unicode in case the format parameters from the context are (might just make sense to add that to the test suite?)

Reply:
It's always unicode since this is coming from the django tokenizer.

Reply:
I guess the weird case would be if you're also using a variable as the format string itself. Like:

```
{% absolute_uri fmt foo bar %}
```

And `fmt` was a variable with something. But we don't use this style anywhere, and again, would have also broken in current version.

Reply:
Actually, I take this back. I found one place we sorta do this. Lemme submit another PR to address it and add tests to cover this.